db/package.js

```diff
@@ -105,11 +110,14 @@ class PackageDbController {
     return { id, ...rest }
   }
 
-  async update ({ packageId, maintainers }) {
+  async update ({ packageId, maintainers, owner }) {
     return this.db.collection('packages').updateOne({
       _id: ObjectId(packageId)
     }, {
-      $set: { maintainers }
+      $set: {
+        ...(maintainers && { maintainers }),
+        ...(owner && { owner })
+      }
     })
   }
```

Comment:
I thought we decided to not have an explicit "owner" ? There are people who are authorized via registry and people authorized via invite. What is the purpose of the owner field?

Reply:
owner determines who can edit the percentages, or were we going off whoever is "source registry" ? doesn't matter to me, i think both will have challenges / benefits

Reply:
i'll enumerate them:

---- having an owner field:
* one person in charge of percentages 
* when project adds more people to registry, they probably aren't thinking about flossbank rights so would protect against surprises there
* probably mirrors most payout systems, we could ask open collective what they do but i assume there is an admin of an OC project who controls payouts.
* downside of having potentially out of date owner who has stopped maintaining the project

--- not having an owner field
* provides easy maintenance for whenever an old maintainer leaves the project, anyone who has source registry can update the revenue share 
* unless we email everyone that rev shares have changed, how will teams know that someone changed flossbank rev shares?
* allows us to easily mirror registry information

Reply:
removed owners field - re-review :)